app/components/flag-subject-button.cjsx

```diff
@@ -0,0 +1,35 @@
+React = require 'react'
+
+module?.exports = React.createClass
+  displayName: 'FlagSubjectButton'
+
+  propTypes:
+    # not sure if we'll need all these
+    subject: React.PropTypes.object # a subject response from panoptes
+    project: React.PropTypes.object # a project response from panoptes
+    workflow: React.PropTypes.object
+    classification: React.PropTypes.object
+    user: React.PropTypes.object
+
+  getInitialState: ->
+    flagged: false
```

Comment:
No need to keep track of this in local component state. It'll be available at `@props.classification.subject_flagged`.